integration-tests/modules/__tests__/cart/store/carts.spec.ts

```diff
@@ -316,6 +316,49 @@ medusaIntegrationTestRunner({
           )
```

Comment:
These are all tests of the HTTP layer, so I'll move these to `integration-tests/http` before merging. Keeping them as is, because the diff would otherwise be off